tests/kernel/boot_ltp.pm

```diff
@@ -32,7 +33,12 @@ sub run {
         $self->wait_boot(ready_time => 500);
     }
 
-    $self->select_serial_terminal;
+    if (check_var('BACKEND', 'ipmi')) {
+        use_ssh_serial_console;
+    }
+    else {
+        $self->select_serial_terminal;
+    }
```

Comment:
@okurz yes only select_serial_terminal would be better, but right now it does not change serialdev http://10.100.12.155/tests/9454

Reply:
Yes, sure. I meant to *extend* `select_serial_terminal` to do exactly that